src/ec/packages/ec-component-timeline/docs/timeline.md

```diff
@@ -0,0 +1,48 @@
+# ECL Twig - EC Timeline
+
+npm package: `@ecl-twig/ec-component-timeline`
+
+```shell
+npm install --save @ecl-twig/ec-component-timeline
+```
+
+## Timeline
+
+### Parameters
+
+- "toggleCollapsed": (string) (default: 'Show %d more items')
+- "toggleExpanded": (string) (default: 'Show %d items')
+- "hide" (array): (default: undefined)
+  - "from": (integer) (default: undefined) Positive integer
+  - "to": (integer) (default: undefined) Negative integer
+- "items" (array) (default: []):
+  - "id": (string) (default: '')
+  - "label": (string) (default: '')
+  - "title": (string) (default: '')
+  - "content": (block) (default: '')
+- "icon_path" (string) (default: ''): file containing the svg icons
+- "extra_classes" (string) (default: '')
+- "extra_attributes" (array) (default: []): format:
+  - "name" (string) (default: '')
+  - "value" (string) (default: '')
+
+### Example :
+
+<!-- prettier-ignore -->
+```twig
+{% include 'path/to/timeline.html.twig' with {  
+  toggleCollapsed: 'Show %d more items', 
+  toggleExpanded: 'Hide %d items', 
+  hide: {from: 7, to: -2}, 
+  items: [ 
+    { 
+      id: '0', 
```

Comment:
`id: '0'` is quite bad, can you change it here? (we will update the data from the specs later)